sdk/security/keyvault/internal/ci.keyvault.yml

```diff
@@ -0,0 +1,28 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+    - sdk/security/keyvault/internal
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+    - sdk/security/keyvault/internal
+
+stages:
+- template: /eng/pipelines/templates/jobs/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: 'keyvault/internal'
```

Comment:
I think this needs to be `security/keyvault/internal`

Reply:
Me too! I'm sure it was user error but I'll never trust find/replace again :suspect: